packages/angular-material/src/module.ts

```diff
@@ -110,7 +114,8 @@ import {
     LabelRenderer,
     CategorizationTabLayoutRenderer,
     MasterListComponent,
-    JsonFormsDetailComponent
+    JsonFormsDetailComponent,
+    ObjectControlRenderer
   ],
   schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
Off-topic, but do we still need `CUSTOM_ELEMENTS_SCHEMA`  to be registered?

Reply:
I would recheck after this is merged.